conda/recipes/cuxfilter/meta.yaml

```diff
@@ -40,7 +40,6 @@ requirements:
     - numpy >=1.23,<2.0a0
     - packaging
     - panel >=1.0
-    - pyppeteer >=0.2.6
```

Comment:
This dependency on `pyppeteer` makes it impossible to install `cuxfilter` in a Python 3.12 environment.

I just pushed https://github.com/rapidsai/cuxfilter/pull/625/commits/7d173772b27cec39e576ce2fa05043194e3b0a12 removing it.

It appears that the latest version of `pyppeteer` ([v1.0.2, published 2.5+ years ago](https://anaconda.org/conda-forge/pyppeteer/files)), puts an upper limit on `websockets<11.0`  and there are not any `websockets` conda packages available that both support Python 3.12 and meet that constraint.

```text
The following packages are incompatible
└─ cuxfilter is installable with the potential options
   ├─ cuxfilter 24.10.00a21 would require
   │  └─ pyppeteer >=0.2.6  with the potential options
   │     ├─ pyppeteer 0.2.6 would require
   │     │  └─ websockets >=9.1,<10.0  with the potential options
   │     │     ├─ websockets [10.0|10.1|10.2|10.3|9.1] would require
   │     │     │  └─ python >=3.7,<3.8.0a0 , which can be installed;
   │     │     ├─ websockets [10.0|10.1|...|9.1] would require
   │     │     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
   │     │     ├─ websockets [10.0|10.1|...|9.1] would require
   │     │     │  └─ python >=3.9,<3.10.0a0 , which can be installed;
   │     │     └─ websockets 9.1 would require
   │     │        └─ python >=3.6,<3.7.0a0 , which can be installed;
   │     └─ pyppeteer 1.0.2 would require
   │        └─ websockets >=10.0,<11.0  with the potential options
   │           ├─ websockets [10.0|10.1|10.2|10.3|10.4] would require
   │           │  └─ python >=3.10,<3.11.0a0 , which can be installed;
   │           ├─ websockets [10.0|10.1|10.2|10.3|9.1], which can be installed (as previously explained);
   │           ├─ websockets [10.0|10.1|...|9.1], which can be installed (as previously explained);
   │           ├─ websockets [10.0|10.1|...|9.1], which can be installed (as previously explained);
   │           └─ websockets 10.4 would require
   │              └─ python >=3.11,<3.12.0a0 , which can be installed;
```

([build link](https://github.com/rapidsai/cuxfilter/actions/runs/10778331677/job/29889656995?pr=625#step:7:395))

I think we can drop the `pyppeteer` dependency here!

* `git grep -i pyppeteer` shows 0 direct uses of it
* it seems the last use of it here was removed when `cuxfilter.assets.screengrab` was removed in #494
* `pyppeteer` appears to be abandoned: https://github.com/pyppeteer/pyppeteer/blob/7dc91ee5173d3836f77800a3774beeaf2b448c0e/README.md#L1